src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.targets

```diff
@@ -45,54 +45,24 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
   <PropertyGroup Condition="'$(PackProjectInputFile)' == ''">
     <PackProjectInputFile>$(MSBuildProjectFullPath)</PackProjectInputFile>
-  </PropertyGroup>    
-
-  <!--
-    Always reference FSharp.core for coreclr builds
-    TargetFSharpCorePackageIdentity = Name of package to reference      (Default = Microsoft.FSharp.Core.netcore)
-    TargetFSharpCorePackageVersion  = Version of package to reference   (Default = TargetFramework = netcoreapp1.0-1.1 netstandard1.0-1.6 then '1.0.0-*')
-                                                                        (Default = TargetFramework = netcoreapp2.0 netstandard2.0         then '1.0.0-*')
-  -->
-  <PropertyGroup Condition=" '$(TargetFSharpCorePackageVersion)' == '' ">
-    <TargetFSharpCorePackageVersion Condition=" $(TargetFramework.StartsWith(netcoreapp1.)) or $(TargetFramework.StartsWith(netstandard1.)) ">1.0.0-*</TargetFSharpCorePackageVersion>
-    <TargetFSharpCorePackageVersion Condition=" $(TargetFramework.StartsWith(netcoreapp2.)) or $(TargetFramework.StartsWith(netstandard2.)) ">1.0.0-*</TargetFSharpCorePackageVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" '$(TargetFSharpCorePackageIdentity)' == '' ">
-    <TargetFSharpCorePackageIdentity Condition=" '$(TargetFrameworkIdentifier)' != '.NETFramework' " >Microsoft.FSharp.Core.netcore</TargetFSharpCorePackageIdentity>
+  <PropertyGroup Condition=" '$(TargetFSharpCorePackageVersion)' == '' ">
+    <TargetFSharpCorePackageVersion>4.2.0-*</TargetFSharpCorePackageVersion>
```

Comment:
Lets just set it to 4.2.* for now. It's not a bad plan.  the -pre-release thing is probably not appropriate to FSharp.Core anyway.